spec/models/hmcts_common_platform/offence_spec.rb

```diff
@@ -69,9 +69,13 @@
       expect(offence.plea).to be_a(HmctsCommonPlatform::Plea)
     end
 
-    it "has a verdict offence id" do
+    it "has a verdict" do
       expect(offence.verdict).to be_a(HmctsCommonPlatform::Verdict)
     end
+
+    it "has an LAA reference" do
```

Comment:
idk if this is ever called like this - if not can we remove this test and make the method private?